docs/guides/elasticsearch/concepts/elasticsearch-dashboard/index.md

```diff
@@ -0,0 +1,103 @@
+    identifier: es-dashboard-concepts
+    name: ElasticsearchDashboard
+    parent: es-concepts-elasticsearch
+    weight: 21
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+# ElasticsearchDashboard
+
+## What is ElasticsearchDashboard
+
+`ElasticsearchDashboard` is a Kubernetes `Custom Resource Definitions` (CRD). It provides a declarative configuration to specify the docker images to be used for Elasticsearch Dashboard (`Kibana`, `Opensearch_Dashboards`) deployed with KubeDB in Kubernetes native way. When you install KubeDB, an `ElasticsearchVersion` custom resource will be created automatically for every supported `ElasticsearchDashboard` version.
+Suppose you have a KubeDB-managed [Elasticsearch](/docs/guides/elasticsearch/concepts/elasticsearch/index.md) provisioned in your cluster. You have to specify the name of `Elasticsearch` CRD in `spec.databaseRef.name` field of `ElasticsearchDashboard` CRD. Then, KubeDB will use the docker images specified in the `ElasticsearchVersion` CRD to create your expected dashboard.
+
+
+## ElasticsearchDashboard Specification
+
+As with all other Kubernetes objects, an `ElasticsearchDashboard` needs `apiVersion`, `kind`, and `metadata` fields. It also needs a `spec` section.
+
+```yaml
+apiVersion: dashboard.kubedb.com/v1alpha1
+kind: ElasticsearchDashboard
+metadata:
+  name: es-cluster-dashboard
+  namespace: demo
+spec:
+  replicas: 1
+  enableSSL: true
+  authSecret:
+    name: es-cluster-user-cred
+  configSecret:
+    name: custom-configuration
+  databaseRef:
+    name: es-cluster
+  podTemplate:
+    spec:
+     resources:
+        limits:
+          memory: 1.5Gi
+        requests:
+          cpu: 500m
+          memory: 1.5Gi
+  serviceTemplates:
+    - alias: primary
+      spec:
+        ports:
+          - port: 5601
+  tls:
+    certificates:
+      - alias: database-client
+        secretName: es-cluster-client-cert
+  terminationPolicy: WipeOut
+```
+
+### spec.enableSSL
+
+`spec.enableSSL` is an `optional` field that specifies whether to enable TLS to HTTP layer. The default value of this field is `false`.
+
+### spec.authSecret
+
+`spec.authSecret` is an `optional` field that points to a k8s secret used to hold the Elasticsearch `elastic`/`admin` user credentials.
+The k8s secret must be of type: kubernetes.io/basic-auth with the following keys:
+
+- `username`: Must be `elastic` for x-pack, and `admin` for `OpenSearch`.
+- `password`: Password for the `elastic`/`admin` user.
+If not set, the KubeDB operator creates a new Secret {Elasticsearch name}-{UserName}-cred with randomly generated secured credentials.
+
+### spec.configSecret
+
+`spec.configSecret` is an optional field that allows users to provide custom configuration for `ElasticsearchDashboard`. It contains a k8s secret name that holds the configuration files for `ElasticsearchDashboard`. The configuration file names are used as secret keys.
```

Comment:
If configSecret is provided, it will be merged with the operator-generated configuration. The user-provided configuration has higher precedence over the operator-generated configuration.